src/elements/category/src/index.tsx

```diff
@@ -16,17 +17,34 @@ const DefaultContainer: React.FC = ({ children }) => (
 const Category: React.FC<ListProps> = ({
   title,
   text,
-  container: Container = DefaultContainer
+  container: Container = DefaultContainer,
+  breadcrumbs: Breadcrumbs
 }) => {
+  const { theme }: any = useThemeUI()
+  const breadcrumbsVariant = theme.categoryTitle?.breadcrumbs?.variant
+
   return (
     <div
       sx={{
         paddingX: ['sm', 'md'],
-        paddingY: ['md', 'lg'],
+        paddingBottom: ['md', 'lg'],
+        paddingTop: ['xs', 'sm'],
```

Comment:
Does this reduce the padding when breadcrumbs aren't specified?

Reply:
Ah yeah, I will fix that